src/main/java/io/turntabl/empireJob/EmpireJobApplication.java

```diff
@@ -17,24 +16,21 @@
 import java.util.Arrays;
 import java.util.List;
 
-
 @EnableSwagger2
 @SpringBootApplication
 public class EmpireJobApplication {
 	@Autowired
 	JdbcTemplate template;
 
 	public static void main(String[] args) throws URISyntaxException, IOException, InterruptedException {
-		String location = "http://192.168.8.122:8050/api/v1/endpoints";
+		String endpoints_location = System.getenv("ENDPOINTS");
 		HttpClient client = HttpClient.newBuilder().build();
 		URI url;
-		url = new URI(location);
+		url = new URI(endpoints_location);
```

Comment:
oh this is the endpoint endpoint. the naming is going to be a little difficult here but can you try to think of something a bit clearer please?